internal/provider/resource_basistheory_proxy.go

```diff
@@ -132,8 +131,12 @@ func resourceProxyCreate(ctx context.Context, data *schema.ResourceData, meta in
 	proxyRequest := *basistheory.NewCreateProxyRequest(proxy.GetName(), proxy.GetDestinationUrl())
 	proxyRequest.SetRequestReactorId(proxy.GetRequestReactorId())
 	proxyRequest.SetResponseReactorId(proxy.GetResponseReactorId())
-	proxyRequest.SetRequestTransform(proxy.GetRequestTransform())
-	proxyRequest.SetResponseTransform(proxy.GetResponseTransform())
+	if proxy.RequestTransform != nil {
+		proxyRequest.SetRequestTransform(proxy.GetRequestTransform())
+	}
+	if proxy.ResponseTransform != nil {
+		proxyRequest.SetResponseTransform(proxy.GetResponseTransform())
+	}
```

Comment:
The Go SDK creates an empty Transform object in the `GetResponseTransform()` method. That will cause validation errors in Vault.

Reply:
You predicted the future! 😂